src/hooks/useStatistics.jsx

```diff
@@ -0,0 +1,21 @@
+import { useQuery } from '@tanstack/react-query';
+import { getStatistics } from '../services/StatisticsService';
+
+export function useStatistics() {
+  const { data, error, isLoading, refetch } = useQuery({
+    queryKey: ['statistics'],
+    queryFn: () => getStatistics(),
+    staleTime: 5 * 60 * 1000,
+    retry: 1,
+    onError: error => {
```

Comment:
I think we don't need one more `console.error` here

Reply:
Attention! Thank you for your attention!